Explorer/Assets/DCL/SDKComponents/Tween/Components/CustomTweener/ICustomTweener.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using CRDT;
+using DCL.ECSComponents;
+using DG.Tweening;
+using UnityEngine;
+
+namespace DCL.SDKComponents.Tween.Components
+{
+    public interface ICustomTweener : IDisposable
+    {
+        public TweenResult GetResult();
+        public CRDTEntity ParentId { get; set; }
```

Comment:
I followed the usages and it looks like this field is never read

Reply:
Not following this one. Here is saved to be passed later to the CRDT
![image](https://github.com/decentraland/unity-explorer/assets/1999557/77e5c311-b414-46b5-81ee-0df153165fa9)